src/gluonts/model/seq2seq/_forking_estimator.py

```diff
@@ -131,9 +131,9 @@ def __init__(
         use_feat_static_cat: bool = False,
         cardinality: List[int] = None,
         embedding_dimension: List[int] = None,
-        add_time_feature: bool = True,
-        add_age_feature: bool = True,
-        enable_decoder_dynamic_feature: bool = True,
+        add_time_feature: bool = False,
+        add_age_feature: bool = False,
+        enable_decoder_dynamic_feature: bool = False,
```

Comment:
Why disabling these by default?

Reply:
So they are by default disabled in MQCNN. This is just to make this consistent.